extensions/amp-story-shopping/0.1/amp-story-shopping-config.js

```diff
@@ -14,6 +17,108 @@ import {
  */
 let ShoppingConfigResponseDef;
 
+/**
+ * Validates string length of shopping config attributes
+ * @param {?string} field
+ * @param {?string} value
+ * @return {!Array<string>}
+ * @private
```

Comment:
I think that static functions don't have to be annotated with `@private` since they are implicitly private (i.e., since they are not exported via `export`, they are considered local and not seen elsewhere). This goes for the other `validate` functions too

Reply:
Sounds good!